reproductions/Directory.Build.props

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <Platforms>x64;x86</Platforms>
     <PlatformTarget>$(Platform)</PlatformTarget>
-    <RuntimeIdentifiers Condition="'$(TargetFramework)' == 'netcoreapp2.1'">win-x64;win-x86;linux-x64</RuntimeIdentifiers>
     <RuntimeIdentifier Condition="'$(TargetFramework)' != 'netcoreapp2.1'">win-$(Platform)</RuntimeIdentifier>
```

Comment:
We should be able to remove them all (both `<RuntimeIdentifiers>` and `<RuntimeIdentifier>`), as long as we never build with SDK 2.0.

I just ran a few builds locally removing them on _all_ files in the solution and build succeeded on both Windows and Linux.